apps/web/src/app/challenge/_components/comments/comment.tsx

```diff
@@ -490,22 +561,41 @@ function ExpandableContent({ content }: { content: string }) {
   return (
     <div
       className={clsx(
-        { 'h-full': expanded, 'max-h-[300px]': !expanded },
-        'relative w-full overflow-hidden break-words pl-[1px] text-sm',
+        { 'h-full': expanded, 'max-h-[100px]': !expanded },
+        'relative w-full overflow-hidden break-words pl-[1px] text-sm transition-transform',
       )}
       ref={contentWrapperRef}
     >
       <Markdown>{content}</Markdown>
       {!expanded && (
         <div
           className="absolute top-0 flex h-full w-full cursor-pointer items-end bg-gradient-to-b from-transparent to-white dark:to-zinc-800"
-          onClick={() => setExpanded(true)}
+          onClick={() => {
+            setExpanded(true);
+            setSeeCollapse(true);
+          }}
         >
           <div className="text-md text-label-1 dark:text-dark-label-1 flex w-full items-center justify-center hover:bg-transparent">
             Read more
           </div>
         </div>
       )}
+      {expanded && seeCollapse && (
```

Comment:
do we actually need a new state `collapse` here. I think `expanded` does  something similar.

If we need a new state, shouldn't this condition be `expanded || seeCollapse` to keep the previous behaviour intact ?